kolibri/plugins/user/assets/src/views/FacilitySelect.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+
+  <AuthBase>
+    <div class="facility-select">
+      <KRouterLink :to="backTo" :text="coreString('goBackAction')" icon="back" />
+      <div>
+        <p>{{ label }}</p>
+        <div v-for="facility in facilityList['enabled']" :key="facility.id" class="facility-name">
+          <KButton appearance="basic-link" @click="setFacility(facility.id)">
+            {{ facility.name }}
+          </KButton>
+        </div>
+      </div>
+      <div v-if="facilityList['disabled'].length" class="disabled-facilities">
+        <p>{{ $tr('askAdminForAccountLabel') }}</p>
+        <div
+          v-for="facility in facilityList['disabled']"
+          :key="facility.id"
+          class="disabled facility-name"
+        >
+          {{ facility.name }}
+        </div>
+      </div>
+    </div>
+  </AuthBase>
+
+</template>
+
+
+<script>
+
+  import { mapGetters } from 'vuex';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import partition from 'lodash/partition';
+  import { PageNames } from '../constants';
+  import AuthBase from './AuthBase';
+
+  export default {
+    name: 'FacilitySelect',
+    components: { AuthBase },
+    mixins: [commonCoreStrings],
+    computed: {
+      ...mapGetters(['facilities']),
+      backTo() {
+        return this.$router.getRoute(this.$route.query.backTo);
+      },
+      facilityList() {
+        if (this.$route.query.next === PageNames.SIGN_UP) {
+          const partitionedFacilities = partition(
+            this.facilities,
+            f => f.dataset.learner_can_sign_up
+          );
+          return {
+            enabled: partitionedFacilities[0],
+            disabled: partitionedFacilities[1],
+          };
+        } else {
+          return { enabled: this.facilities, disabled: [] };
+        }
+      },
+      label() {
+        return this.$route.query.next === PageNames.SIGN_UP
+          ? this.$tr('canSignUpForFacilityLabel')
+          : this.$tr('selectFacilityLabel');
+      },
+    },
+    methods: {
+      setFacility(facilityId) {
+        console.log(`setting facId ${facilityId}`);
```

Comment:
Should remove this console log

Reply:
I wonder why linting didn't get mad about that, thanks!